fdbserver/fdbserver.actor.cpp

```diff
@@ -1374,29 +1371,28 @@ int main(int argc, char* argv[]) {
 
 		if (seedConnString.length() && seedConnFile.length()) {
 			fprintf(stderr, "%s\n", "--seed_cluster_file and --seed_connection_string may not both be specified at once.");
-			return FDB_EXIT_ERROR;
+			flushAndExit(FDB_EXIT_ERROR);
```

Comment:
I don't think this will work here as the trace file has not yet been opened.

Reply:
I think `flushAndExit` is still safe to call if the trace file has not yet been opened